README.md

```diff
@@ -144,10 +144,12 @@ duplicate([1,2,3,4,5]); // [1,2,3,4,5,1,2,3,4,5]
 * What are some of the advantages/disadvantages of writing JavaScript code in a language that compiles to JavaScript?
 * What tools and techniques do you use debugging JavaScript code?
 * What language constructions do you use for iterating over object properties and array items?
+* Explain destructuring assignment.
 * Explain the difference between mutable and immutable objects.
   * What is an example of an immutable object in JavaScript?
   * What are the pros and cons of immutability?
   * How can you achieve immutability in your own code?
+* What is a pure function? What are the side effects of a pure function?
```

Comment:
Kinda. I guess it could be reworded to be something like:

"Does a pure function have any side effects?"